Directory.Build.props

```diff
@@ -20,8 +20,10 @@ tools\TestAssetsDependencies\TestAssetsDependencies.csproj
 
     <NuGetPackagesDir>$(NUGET_PACKAGES)</NuGetPackagesDir>
     <NuGetPackagesDir Condition=" '$(NuGetPackagesDir)' == '' ">$(RepoRoot)/.nuget/packages</NuGetPackagesDir>
+    <GeneratedNuGetConfig>$(RepoRoot)/NuGet.Config</GeneratedNuGetConfig>
```

Comment:
This line seems out of place - "GeneratedNuGetConfig" - if the branch has a static 'Nuget.config' at the root how is this value being used?

Reply:
This is setting the path to the repo root nuget.config that is generated. Had to move here because we don't have a two pass build in this branch. Plus, this depends on nothing from the two pass.